### ISSUE TITLE
Still in development?

### ISSUE THREAD

**Issue by Larx** [brantje/passman#123]
As the coding activity seems to stagnate, one question: Is Passman still in development, or is it time to switch to another OC password management app (what I would not like to do)?

**Comment by animalillo** [comment]
Yes we are still working on passman, but we have a life outside the project, you know.

I'm working on a new sharing feature when I can.

**Comment by brantje** [comment]
As @animalillo stated we have a personal life too.
Currently i have a 40 hour job and some other projects too.

**Comment by Larx** [comment]
I didn't want to insult anybody, as this is free software, however, with another password manager being actively updated in the owncloud app store and this product seemingly stagnating, I just wanted to ask whether it is time to do the switch.

I'm running the git version of passman since a few weeks, which works very good IMHO, but it's not official...

**Comment by brantje** [comment]
The git version is the most stable at the moment, i'm considering releasing it with out sharing.
The sharing part will come later then.

**Comment by ib54003** [comment]
any news?

**Comment by brantje** [comment]
Yea, this weekend i have time for passman =)

**Comment by ib54003** [comment]
perfect ;)

**Comment by animalillo** [comment]
Yes, I also have more free time now to work on passman, so progress in the sharing branch will be made in the near future if nothing comes up in the way again

**Comment by ib54003** [comment]
Great, especially password sharing would be awesome!

**Comment by animalillo** [comment]
@ib54003 Here you can follow up sharing progress #131

**Comment by ib54003** [comment]
@animalillo @brantje still alive?

**Comment by animalillo** [comment]
@ib54003 yes, we are still alive, and no, passman is not dead. But you know, we have lives and the thing just works nicely and smoothly, right?

Btw, for your delight: An update and full app rewrite of Passman for NextCloud is in progress, follow up [in the NexCloud repo](https://github.com/nextcloud/passman)

**Comment by animalillo** [comment]
My feelings regarding this kind of issues
![my feelings](http://www.commitstrip.com/wp-content/uploads/2014/05/Strip-Vision-Open-source-650-finalenglish.jpg)
